Modelica/Clocked/Examples/SimpleControlledDrive/ClockedWithDiscretizedContinuousController.mo

```diff
@@ -28,8 +28,8 @@ model ClockedWithDiscretizedContinuousController
   Clocked.RealSignals.Sampler.Sample sample1
     annotation (Placement(transformation(extent={{30,-26},{18,-14}})));
   Modelica.Blocks.Continuous.PI PI(
-    initType=Modelica.Blocks.Types.Init.InitialOutput,
-    y_start=0,
+    initType=Modelica.Blocks.Types.Init.NoInit,
+    x(fixed=true),
```

Comment:
Isn't `x(fixed = true)` a violation of this rule from https://specification.modelica.org/master/synchronous-language-elements.html#initialization-of-clocked-partitions?
> Attribute `fixed` cannot be applied to variables in clocked partitions. The attribute `fixed` is true for variables to which `previous` is applied, otherwise false.

Reply:
Ouch. So, how are initial conditions specified for such clocked partitions?

Reply:
That's a valid question, but more work on the specification side is needed to answer it.  Until then, I would just find it best to avoid breaking one of the few simple rules we have.

By the way, System Modeler also warns about underdetermined initialization where `PI.x` was automatically selected to be initialized from `start`, so it is not only in Dymola that this example will not run as cleanly as we'd ideally want it to.

Reply:
I should also mention that System Modeler is lacking a check for
> Attribute `fixed` cannot be applied to variables in clocked partitions.

so I can't tell how close the rest of the MSL is to complying with the rule.

Reply:
The "having fully defined initial conditions" is a strong requirement for tool-independent behaviour. If this is not possible with the current specification, I agree at relaxing/changing it. @HansOlsson would the proposed changes to the code OK with your proposed change to the MLS?

Reply:
With the proposed change to the specification there is *no need to update this model at all*, as the restrictions apply equally to initial equations and fixed=true.